src/common/components/vanilla/button/button-positive.css

```diff
@@ -0,0 +1,29 @@
+$button-color: $success;
```

Comment:
@katiefenn but if smaller components share most of the code and differ only in style it would be good to have one implementation (with possible switches) and exports of different components with those switches set to different values. What I mean is to have `Button` that can be anything (with some style attribute), but we may also have `ButtonPositive` that sets the style to some specific value.